src/components/PassengerShowcaseForm.js

```diff
@@ -238,9 +238,24 @@ const PassengerShowcaseForm = () => {
           Video
           <select name="video" value={state.video} onChange={onChange}>
             <option value="">Select a video</option>
-            {(
-              data.find((track) => track.slug === state.track)?.videos || []
-            ).map((node) => {
+
+            {(selectedTrack?.chapters || []).map((node) => (
```

Comment:
You could just continue the optional chain here, a little nicer than the empty array imo. Same on line 258.

`selectedTrack?.chapters?.map(...)`